retrieval/scrape.go

```diff
@@ -285,13 +285,15 @@ type targetScraper struct {
 }
 
 const acceptHeader = `application/vnd.google.protobuf;proto=io.prometheus.client.MetricFamily;encoding=delimited;q=0.7,text/plain;version=0.0.4;q=0.3,*/*;q=0.1`
+const userAgentHeader = `Prometheus-scraper/1.0`
```

Comment:
Browsers include both the engine/version as well as browser/version where the latter includes the exact software build version. As we haven't versioned the scraper component of the Prometheus server so far, I'd not try to invent now.

Given the version number describes the Prometheus server and not the scraper component, I'd drop the `-scraper` part and just use `Prometheus/1.5.2`.